src/infrastructure/database/prisma/repositories/prisma.space-invitation.repository.ts

```diff
@@ -52,6 +54,26 @@ export class PrismaSpaceInvitationRepository
     return this.toSpaceInvitation(prismaInvitation);
   }
 
+  async findValidSpaceInvitation(
```

Comment:
こっちのほうが他のメソッドど統一性がとれるかな
```suggestion
  async findSpaceInvitationsBySpaceId(
```